extension/error.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  EINVAL: 'EINVAL', // validation code
+  EAUTH: 'EAUTH',
```

Comment:
EAUTH is an internal error (related to the backend). It should not be exposed. Instead an EINTERNAL code should be exposed, that it's clear that the caller can't do anything about it.

Reply:
done